src/mongocxx/options/find_one_and_replace.hpp

```diff
@@ -19,9 +19,11 @@
 #include <cstdint>
 
 #include <bsoncxx/document/view.hpp>
-#include <mongocxx/options/find_one_and_modify.hpp>
 #include <bsoncxx/stdx/optional.hpp>
```

Comment:
Do we still need this now?  Should headers that use stdx::optional include <mongocxx/stdx.hpp>, <bsoncxx/stdx/optional.hpp>, or both?